sdk/resourcemanager/Azure.ResourceManager.Core/src/Generated/TagsContainer.cs

```diff
@@ -0,0 +1,155 @@
+        public virtual Response<TagDetails> CreateOrUpdate(string tagName, CancellationToken cancellationToken = default)
+        {
+            using var scope = Diagnostics.CreateScope("TagsOperations.CreateOrUpdate");
+            scope.Start();
+            try
+            {
+                return RestClient.CreateOrUpdate(tagName, cancellationToken);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary> This operation performs a union of predefined tags, resource tags, resource group tags and subscription tags, and returns a summary of usage for each tag name and value under the given subscription. In case of a large number of tags, this operation may return a previously cached result. </summary>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        public virtual AsyncPageable<TagDetails> ListAsync(CancellationToken cancellationToken = default)
+        {
+            async Task<Page<TagDetails>> FirstPageFunc(int? pageSizeHint)
+            {
+                using var scope0 = Diagnostics.CreateScope("TagsOperations.List");
+                scope0.Start();
+                try
+                {
+                    var response = await RestClient.ListAsync(cancellationToken).ConfigureAwait(false);
+                    return Page.FromValues(response.Value.Value, response.Value.NextLink, response.GetRawResponse());
+                }
+                catch (Exception e)
+                {
+                    scope0.Failed(e);
+                    throw;
+                }
+            }
+            async Task<Page<TagDetails>> NextPageFunc(string nextLink, int? pageSizeHint)
+            {
+                using var scope0 = Diagnostics.CreateScope("TagsOperations.List");
+                scope0.Start();
+                try
+                {
+                    var response = await RestClient.ListNextPageAsync(nextLink, cancellationToken).ConfigureAwait(false);
+                    return Page.FromValues(response.Value.Value, response.Value.NextLink, response.GetRawResponse());
+                }
+                catch (Exception e)
+                {
+                    scope0.Failed(e);
+                    throw;
+                }
+            }
+            return PageableHelpers.CreateAsyncEnumerable(FirstPageFunc, NextPageFunc);
+        }
+
+        /// <summary> This operation performs a union of predefined tags, resource tags, resource group tags and subscription tags, and returns a summary of usage for each tag name and value under the given subscription. In case of a large number of tags, this operation may return a previously cached result. </summary>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        public virtual Pageable<TagDetails> List(CancellationToken cancellationToken = default)
```

Comment:
should return the full [Resource] not the [Resource]Data

Reply:
Done